### PR TITLE
Flags to specify path of adb

### PR DESCRIPTION
**Pull request by pmuetschard**
For #525, #478, and #14

### CODE REVIEW

**Comment by ben-clayton** [comment]
Build fails:

```
../../github/src/github.com/google/gapid/cmd/gapit/packages.go:47: undefined: adb in adb.ADB
../../github/src/github.com/google/gapid/cmd/gapit/packages.go:47: undefined: file in file.Abs
../../github/src/github.com/google/gapid/cmd/gapit/packages.go:47: cannot assign to adb.ADB
```

**Comment by pmuetschard** [comment]
> Build fails:

Fixed.

File: cmd/gapit/flags.go [hunk 1]
```diff
@@ -201,5 +202,6 @@ type (
 		Format      PackagesOutput `help:"output format"`
 		Out         string         `help:"output file, standard output if none"`
 		DataHeader  string         `help:"marker to write before package data"`
+		ADB string `help: "Path to the adb executable; leave empty to search the environment"`
```
Comment (ben-clayton) [inline cmd/gapit/flags.go]: gofmt

Comment (pmuetschard) [inline cmd/gapit/flags.go]: done.

**Pull request merged**